src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy.mdx

```diff
@@ -136,3 +136,134 @@ The policies index lists policies in alphabetical order. To view or search for e
 <Callout variant="tip">
   To view [policy and condition information for a specific entity](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/view-policy-conditions-new-relic-products), select the **Settings > Alert conditions** page from the entity's product UI.
 </Callout>
+
+## Policy maintenance quick reference [#quick-reference-policies]
+
+After you create a policy, you can:
+
+* Add more conditions to the policy.
+* Edit the conditions' names, targets, or thresholds.
+* Disable conditions.
+* Continue the policy setup process by adding one or more notification channels to it.
+* Rename or delete the policy.
+
+The user interface shows a **Last modified** timestamp for any changes to policies, including their conditions and notification channels.
+
+Here is a quick reference which also includes links to more detailed information and procedures for maintaining your policies.
+
+<CollapserGroup>
+  <Collapser
+    id="maintain-policies-channel"
+    title="Add or remove policies assigned to a channel"
+  >
+    To [add or remove policies](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/add-or-remove-policy-channels) assigned to a channel:
+
+    1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, click **Notification channels**, then **(select a channel)**. Click **Alert policies**, then **(select a policy)**.
```

Comment:
There's a lot of steps here and bullets may be a better format. Not sure about public docs style/standards though

Reply:
Yeah, we tend not to use bullets for steps like this.